ufcc/default.ini

```diff
@@ -0,0 +1,9 @@
+[Parameters]
+cutoff = 7
+lipid_types = POPC, DPPC, DOPC, CHOL, CHL1, POPS, POPE
+backend = serial
+n_jobs = -1
+
+[Report]
+build-report = True
```

Comment:
By default, we should not output a report. The default outputs should be either the CSV or the dashboard (or both).